package.json

```diff
@@ -19,7 +19,7 @@
     "@mui/x-data-grid": "^5.4.0",
     "@reduxjs/toolkit": "^1.7.1",
     "@sentry/nextjs": "^6.17.4",
-    "@superfluid-finance/sdk-core": "0.3.3-dev.4c31a63.0",
+    "@superfluid-finance/sdk-core": "0.4.2-dev.b0d8dd0.0",
```

Comment:
should we use https://www.npmjs.com/package/@superfluid-finance/sdk-core public 0.4.1 instead?

Reply:
ku ku, just to try slack notif

Reply:
Yes, let's use the stable one. There used to be issues with the events table on Subgraph which forced us to "dev" build before.

Reply:
Changed sdk-core package version to latest.